packages/output/src/utils/shoppingAttachment.js

```diff
@@ -18,10 +18,15 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { __ } from '@googleforcreators/i18n';
 
-function ShoppingAttachment({ products, theme }) {
+function ShoppingAttachment({
+  products,
+  theme,
+  ctaText = __('Shop Now', 'web-stories'),
+}) {
   return (
-    <amp-story-shopping-attachment theme={theme}>
+    <amp-story-shopping-attachment theme={theme} cta-text={ctaText}>
```

Comment:
Let's not add a default value for `ctaText` here _and_ let's not add the `cta-text` attribute if the default `__('Shop Now', 'web-stories')` is provided.

Why?

If it's absent, AMP will provide the default "Shop New" text already anyway. Plus, the default text is translated into dozens of locales, which is quite handy.

Reply:
Conditional attribute or empty attriubte?

Reply:
Fixed.